transport-msg.tex

```diff
@@ -737,52 +738,47 @@ \subsubsection{Overview}
 \hline
 \msgref{RESET_VQUEUE}      & 0xB  & Driver \\
 \hline
-\msgref{EVENT_CONFIG}      & 0x20 & Device \\
+\msgref{EVENT_CONFIG0}     & 0x40 & Device \\
+\hline
+\msgref{EVENT_CONFIG}      & 0x41 & Device \\
 \hline
-\msgref{EVENT_AVAIL}       & 0x21 & Driver \\
+\msgref{EVENT_AVAIL}       & 0x42 & Driver \\
 \hline
-\msgref{EVENT_USED}        & 0x22 & Device \\
+\msgref{EVENT_USED}        & 0x43 & Device \\
 \hline
 \end{tabular}
 
+Transport message IDs 0x00 to 0x3F are used for messages that require a response
+and IDs 0x40 to 0x7F are used for messages that have no response.  Transport
```

Comment:
We are using "event" a lot before including the fact that there is not response needed so i would suggest to rephrase here:
0x40 to 0x7F are used for event messages.